fmskill/plot.py

```diff
@@ -173,7 +173,11 @@ def scatter(
         # if not an int nor None, it must be a squence of floats
         xq = np.quantile(x, q=quantiles)
         yq = np.quantile(y, q=quantiles)
-
+    
+    if show_hist:
+        #if histogram is wanted (explicit non-default flag) then density is off
+        show_density=False
```

Comment:
If a user explicitly specifies both show_hist=True and show_density=True this generate an error and not silently be changed.

In order to keep track of  user defined arguments and defaults you can change the default to be show_hist=None

Reply:
Thanks for reviewing. 
* I changed both defaults to `None` and if both are none (no user input) then `density` switches to `True`
* I raised an error if both are explicitly switched to True (user needs to choose if they want density or histogram, not both)
* I fixed the colorbar when `2dhist=True` and `show_points=False`
* Minor change in QQ marker_edge_alpha
* Uploading branch now

![image](https://user-images.githubusercontent.com/97288080/192976139-68ddb870-e5bb-4c9f-9188-8549226d5919.png)
![image](https://user-images.githubusercontent.com/97288080/192976172-54368f74-cce8-4991-a14e-682d754bffcb.png)
![image](https://user-images.githubusercontent.com/97288080/192976186-9ca2f9f6-a687-44be-adca-b1a256bdbd7f.png)
![image](https://user-images.githubusercontent.com/97288080/192976202-d9b3219b-fd40-4954-a71c-8802125cc777.png)
![image](https://user-images.githubusercontent.com/97288080/192976222-3709db60-f4b8-4201-a7d7-bd914014cb7b.png)